62.CAD/main.cpp

```diff
@@ -59,18 +218,31 @@ class Camera2D : public core::IReferenceCounted
 		return m_bounds;
 	}
 
-	float64_t3x3 constructViewProjection()
+	float64_t4x4 constructViewProjection(double timeElapsed)
```

Comment:
I know this is a test, but you could've done it without changing the camera api :D
simply construct a rotation matrix and multiply by the viewProjection outside. 
it's useful to have this test, so make sure when you're done with the test comment it out so we can use it later to test rotation. (but outside this function please)